src/components/activity-page/section.test.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Section } from "./section";
+import { configure, fireEvent, render } from "@testing-library/react";
```

Comment:
minor linting error here: `fireEvent` is never used. Same with `getByText` below.

Reply:
I was hoping to keep the collapsible header test in the unit test. Decided to move it to Cypress test instead.